google-cloud-storage/samples/storage_configure_retries.rb

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START storage_configure_retries]
+def configure_retries bucket_name: nil, file_name: nil
```

Comment:
I realize the sample format you are using here is consistent with the existing samples in this repo. However, we have made some updates to the recommended sample format, and I'd like us to begin using it for new samples, even if it's not consistent with existing samples. (We'll need to update the existing samples over time.)

See https://googlecloudplatform.github.io/samples-style-guide/ for the samples style guide, and see the `google-cloud-secret_manager` samples for examples of the updated format. In particular:

* Move the `require` statement out of the method.
* Document the parameters (such as `bucket_name`) in yardoc-formatted comments just before the method definition, rather than in comments inside the method.
* The method name should match the file name. In most cases, you should start by naming the method according to what would be most reader-friendly for samples. So this existing method name `configure_retries` is good. That means, this file should be named `configure_retries.rb` (instead of `storage_configure_retries.rb`). The file name does _not_ need to match the sample tag comment above.
* Omit the `if $PROGRAM_NAME == __FILE__` clause at the bottom. Samples should not include code that implements a CLI.